system/perses/charts/values.yaml

```diff
@@ -0,0 +1,244 @@
+global:
+  # -- Labels to add to all resources.
+  # This can be used to add a `support_group` or `service` label to all resources and alerting rules.
+  commonLabels: {}
+
+perses:
+  # Default values for perses.
+  # This is a YAML-formatted file.
+  # Declare variables to be passed into your templates.
+
+  # -- Override name of the chart used in Kubernetes object names.
+  nameOverride: ""
+
+  # -- Override fully qualified app name
+  fullnameOverride: ""
+
+  # -- Image of Perses
+  image:
+    # -- Perses image repository and name
+    name: "persesdev/perses"
+
+    # -- Overrides the image tag whose default is the chart appVersion.
+    version: ""
+
+    # -- Default image pull policy
+    pullPolicy: IfNotPresent
+
+  additionalLabels:
+    {}
+    # app: perses
+
+  # -- Service account for Perses to use.
+  serviceAccount:
+    # -- Specifies whether a service account should be created
+    create: true
+    # -- Annotations to add to the service account
+    annotations: {}
+    # -- The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+  # -- Expose the Perses service to be accessed from outside the cluster (LoadBalancer service).
+  # or access it from within the cluster (ClusterIP service). Set the service type and the port to serve it.
+  service:
+    # -- Annotations to add to the service
+    annotations: {}
+    # -- Labeles to add to the service
+    labels:
+      greenhouse.sap/expose: "true"
+    # -- Service Type
+    type: "ClusterIP"
+    # -- Service Port Name
+    portName: http
+    # -- Service Port
+    port: 8080
+    # -- Perses running port
+    targetPort: 8080
+
+  serviceMonitor:
+    # -- Create a serviceMonitor for Perses
+    selfMonitor: false
+
+    # -- Labels to add to the ServiceMonitor so that Prometheus can discover it.
+    # These labels should match the 'serviceMonitorSelector.matchLabels' and `ruleSelector.matchLabels` defined in your Prometheus CR.
+    labels: {}
```

Comment:
I'd go with a default prometheus for our use-case here.

Where is Perses going to be deployed, scaleout? take infra-frontend or infra-collector, depending which cluster you are.

Reply:
The perses is going to be deployed in `metal` clusters. Which prometheus should I take for `metal`? @viennaa

Reply:
`infra-collector` pls